README.rst

```diff
@@ -0,0 +1,105 @@
+Description |BuildStatus|_
```

Comment:
The `|BuildStatus|_` is wrong as it's not pointing at anything. We should probably build and test this project using travis-ci.org for example.